ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -47,6 +74,18 @@ trait StorageBackend[DB_BATCH] {
     * @return the LedgerEnd, which should be the basis for further indexing.
     */
   def initialize(connection: Connection): StorageBackend.LedgerEnd
+}
+
+// TODO append-only: consolidate parameters table facing functions
+trait ParameterStorageBackend {
+
+  /** This method is used to update the parameters table: setting the new observable ledger-end, and other parameters.
+    * No significant CPU load, mostly blocking JDBC communication with the database backend.
+    *
+    * @param connection to be used when updating the parameters table
+    * @param params     the parameters
+    */
+  def updateParams(params: StorageBackend.Params)(connection: Connection): Unit
```

Comment:
I was first surprised that you use the curried form without using implicits, but this form makes it easier to pass these methods to `DbDispatcher.executeSql(Connection => T)` :+1: